### PR TITLE
profiler/getting_started: improve go snippet

### PR DESCRIPTION
**Pull request by felixge**
This came up in a customer call, they hadn't noticed the text added in ee9cb474f889c760a08a8fe694b6bfb390077c4e and suggested to directly include the hint about it in the code snippet.

### Preview

https://docs-staging.datadoghq.com/felix.geisendoerfer/improve-go-profiler-code-snippet/tracing/profiler/getting_started/?tab=go

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by felixge** [comment]
Note to self: There is also some inline docs in the web-ui for APM that includes profiling snippet, we should probably do a similar change there after this gets merged.

File: content/en/tracing/profiler/getting_started.md
Comment (kayayarai) [inline content/en/tracing/profiler/getting_started.md:217]: I agree with adding the sample code comments, but I think this note, particularly the links in it, is useful and shouldn't be removed.

**Review by kayayarai — CHANGES_REQUESTED** [review]

File: content/en/tracing/profiler/getting_started.md
Comment (GaurabAryal) [inline content/en/tracing/profiler/getting_started.md:209]: Thoughts on adding an empty line here for readability?

**Review by GaurabAryal — APPROVED** [review]
LGTM

File: content/en/tracing/profiler/getting_started.md
Comment (felixge) [reply content/en/tracing/profiler/getting_started.md:209]: Sure cbde153510b84b40721b3c825444162a75fe3de8 👍

File: content/en/tracing/profiler/getting_started.md
Comment (felixge) [reply content/en/tracing/profiler/getting_started.md]: @kayayarai ok, I reverted it again for now: 38d765289ffebe88d4f32708baa8fc7802448feb

That being said, I'd love to hear your thoughts on documenting each individual configuration option in this repository vs the [documentation of the package itself](https://pkg.go.dev/gopkg.in/DataDog/dd-trace-go.v1/profiler)?

It's also a little odd that the `WithProfileTypes` option is not listed in the table like the other options we're describing (my fault, not sure why I did this).

I don't think this discussion should hold up this PR, but I'd love to make the Go profiler docs better in general so if you have any thoughts please let me know. I'm happy to do some follow-up PRs.

**Comment by felixge** [comment]
This is ready for another review. Many thanks in advance 🙇‍♂️ !

**Review by kayayarai — APPROVED** [review]

File: content/en/tracing/profiler/getting_started.md
Comment (kayayarai) [reply content/en/tracing/profiler/getting_started.md]: I'm about to go through the Profiling section of the docs and identify gaps and inconsistencies and places we can improve. I'll add this matter to that list. Thank you!

Comment (felixge) [reply content/en/tracing/profiler/getting_started.md]: @kayayarai awesome, thank you so much!

**Pull request merged**
